h/security/permits/permission_map.py

```diff
@@ -0,0 +1,87 @@
+from h.security import Permission
+from h.security.permits.predicates import *
+
+PERMISSION_MAP = {
```

Comment:
This is the separate list way of doing it. Locally very compact, but I think something is lost in maintaining the two things seperately.

Reply:
We talked about this on a call but I think the separate `PERMISSION_MAP` might actually be nicer. It's more straightforward and it separates the permissions themselves from the security policy (an app might want to use different security policies or at least different permission maps in different contexts, but they might all be based on the same set of permissions).